.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       image: ${{ inputs.container-image || 'timescaledev/toolkit-builder:debian-11-amd64' }}
     strategy:
       matrix:
-        pgversion: [12, 13, 14]
+        pgversion: [12, 13, 14, 15]
```

Comment:
We need to break this up into separate merges.
1. Push updated image (#639 also needs to go in, and #621 is harmless and nice to have).
2. This line and all the rest.

Reply:
Never mind; it's not worth the bother to put both 0.5 and 0.6 support into the image, so I'm happy to just push an updated image from this branch before merging it.